src/query/storages/fuse/fuse/src/pruning/pruning_executor.rs

```diff
@@ -79,7 +79,7 @@ impl BlockPruner {
             .and_then(|p| p.limit);
 
         // prepare the limiter. in case that limit is none, an unlimited limiter will be returned
-        let limiter = limiter::new_limiter(limit);
+        let limiter = limiter_pruner::new_limiter(limit);
```

Comment:
I prefer to have a `LimiterPruner::new()` API which carries more context about this struct. `new_limiter` could be anything if imported directly.

Reply:
`LimiterPruner` is also a trait.